src/usr/bin/google_set_multiqueue

```diff
@@ -264,12 +264,21 @@ num_cpus=$(nproc)
 
 num_queues=0
 for q in $XPS; do
+  interface=$(echo "$q" | grep -oP 'net/\K[^/]+')
```

Comment:
This code runs everywhere, not just A3. Are you sure that skipping tx affinity on non-gvnics is the right thing to do on every platform?

Reply:
Sounds good, thanks for the suggestion.